health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/hooks/useUpsertSchemaConfig.js

```
@@ -0,0 +1,31 @@
+import { useMutation } from "react-query";
+
+const upsertSchemaConfig = async (req, tenantId) => {
+  try {
+    debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)